### DISCUSSION TITLE
Query About GSOC 2024

### DISCUSSION THREAD

**Discussion by piyush-1234** [checkstyle/checkstyle#14535] [Q&A] [answered]
hi @romani and @nrmancuso 

my query is 

can I take participation in GSOC 2024 ? 

I am a working professional having 2+ years of experience as a software engineer in another company.

I have been contributing to Checkstyle since Oct 2023 and I am continuing it.
so wanted to know whether i can be a part of GSOC or not.

**Comment by romani** [comment, marked as answer]
Everyone is welcome to participate.
Requirements are described at https://developers.google.com/open-source/gsoc/faq#what_are_the_eligibility_requirements_for_participation

**Reply by piyush-1234** [reply]
Thanx @romani
